Multisig/UI/Transactions/TransactionsList/TransactionsListViewModel.swift

```diff
@@ -19,37 +19,42 @@ struct TransactionsListViewModel {
         }
     }
 
-    private var queued = [TransactionViewModel]()
-    private var history = [TransactionViewModel]()
+    private var allSections: [Section] = []
 
     var sections: [Section] {
-        [Section(name: "QUEUE", transactions: queued),
-         Section(name: "HISTORY", transactions: history)
-        ].filter { !$0.isEmpty }
+        allSections.filter { !$0.isEmpty }
     }
 
     var isEmpty: Bool {
         sections.allSatisfy { $0.isEmpty }
     }
 
     init(_ models: [TransactionViewModel] = []) {
+        var queued = [TransactionViewModel]()
+        var history = [TransactionViewModel]()
+
         for model in models {
-            if model.status.isInQueue {
-                queued.append(model)
-            } else {
-                history.append(model)
-            }
+           if model.status.isInQueue {
+               queued.append(model)
+           } else {
+               history.append(model)
+           }
         }
+
+        allSections = [
+           Section(name: "QUEUE", transactions: queued),
+           Section(name: "HISTORY", transactions: history)
+        ]
     }
 
     mutating func add(_ models: [TransactionViewModel] = []) {
-        for model in models {
-            if model.status.isInQueue {
-                queued.append(model)
-            } else {
-                history.append(model)
-            }
-        }
+       for model in models {
+           if model.status.isInQueue {
+               allSections[0].transactions.append(model)
+           } else {
+               allSections[1].transactions.append(model)
```

Comment:
didn't it caused crash before?

Reply:
`assert(allSections.count > 1)`

Reply:
Please see the init function, previously we used to filter the empty sections and that's why it cased crash, now I'm filtering them only in the getter "sections"